docs/plugins/controller/kubejobs.md

```diff
@@ -61,7 +61,6 @@ In order to execute the plugin, a JSON needs to be correctly configurate with al
       "config_id":"id",
       "control_plugin":"kubejobs",
       "control_parameters":{  
-         "max_size":9,
```

Comment:
unused? this change should be explicitly pointed in the commit description. or done in a separate one.